test/connectivity/NetworkController/LinuxNetworkInterfaceCommands.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class LinuxNetworkInterfaceCommands : INetworkInterfaceCommands
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<LinuxNetworkInterfaceCommands>();
```

Comment:
In LinuxFirewallCommands, network interface name is passed in constructor; but in LinuxNetworkInterfaceCommands, interface name is passed through method.  I think we should use the same pattern and remove interfaceName argument from INetworkInterfaceCommands.